bin/start_backend.sh

```diff
@@ -206,7 +206,7 @@ else
     exec &>> ${LOG_FILE}
 fi
 
-echo "start time: "$(date)
+echo "start time: "$(date)" server:"$(uptime)
```

Comment:
```suggestion
echo "start time: $(date), server:$(uptime)"
```

Reply:
it is ok to include $() inside "", shell will interpret it correctly.

Reply:
done